security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,32 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-30"]
+== OpenShift File Integrity Operator 0.1.30
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.30:
+
+* link:https://access.redhat.com/errata/RHBA-2022:5538[RHBA-2022:5538 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-30-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The File Integrity Operator is now supported on the following architectures:
++
+** IBM Power
+** IBM Z and LinuxONE
+
+[id="file-integrity-operator-0-1-30-bug-fixes"]
+=== Bug fixes
+
+* Previously, the File Integrity Operator used broad permissions coupled with missing labels, causing log warnings about excessive permissions granted to the Operator. Now, permissions are reduced where appropriate and labels are applied to the namespace, eliminating permission warnings in the log files. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088201[*BZ#2088201*])
```

Comment:
I don't think we should document this one, because it was primarily an upstream fix to silence the warnings, and customers normally installing from the marketplace will still see the warnings. 

The actual downstream fix will come later when OLM can adjust our namespace labels for us.